demos/smart_classroom_demo/main.cpp

```diff
@@ -33,15 +35,35 @@ namespace {
 class Visualizer {
 private:
     cv::Mat frame_;
+    cv::Mat top_persons_;
     const bool enabled_;
+    const int num_top_persons_;
     cv::VideoWriter& writer_;
     float rect_scale_x_;
     float rect_scale_y_;
     static int const max_input_width_ = 1920;
-    std::string const window_name_ = "Smart classroom demo";
+    std::string const main_window_name_ = "Smart classroom demo";
+    std::string const top_window_name_ = "Top-k students";
+    static int const crop_width_ = 128;
+    static int const crop_height_ = 320;
+    static int const header_size_ = 80;
+    static int const margin_size_ = 5;
 
 public:
-    Visualizer(bool enabled, cv::VideoWriter& writer) : enabled_(enabled), writer_(writer) {}
+    Visualizer(bool enabled, cv::VideoWriter& writer, int num_top_persons)
+        : enabled_(enabled), writer_(writer), rect_scale_x_(0), rect_scale_y_(0),
+          num_top_persons_(num_top_persons) {
+        if (enabled_) {
```

Comment:
Since the whole body of the fuinction inside this condition, please, negate this condition, and if the negated condition is true, make `return`

Reply:
Done